pytorch_lightning/metrics/functional/classification.py

```diff
@@ -139,39 +139,15 @@ def stat_scores_multiple_classes(
     Calculates the number of true positive, false positive, true negative
     and false negative for each class
 
-    Args:
-        pred: prediction tensor
-        target: target tensor
-        num_classes: number of classes if known
-        argmax_dim: if pred is a tensor of probabilities, this indicates the
-            axis the argmax transformation will be applied over
-        reduction: a method to reduce metric score over labels (default: none)
-            Available reduction methods:
-
-            - elementwise_mean: takes the mean
-            - none: pass array
-            - sum: add elements
-
-    Return:
-        True Positive, False Positive, True Negative, False Negative, Support
-
-    Example:
-
-        >>> x = torch.tensor([1, 2, 3])
-        >>> y = torch.tensor([0, 2, 3])
-        >>> tps, fps, tns, fns, sups = stat_scores_multiple_classes(x, y)
-        >>> tps
-        tensor([0., 0., 1., 1.])
-        >>> fps
-        tensor([0., 1., 0., 0.])
-        >>> tns
-        tensor([2., 2., 2., 2.])
-        >>> fns
-        tensor([1., 0., 0., 0.])
-        >>> sups
-        tensor([1., 0., 1., 1.])
+    .. warning :: Deprecated in favor of :func:`~pytorch_lightning.metrics.functional.stat_scores`
 
     """
+
+    rank_zero_warn(
+        "This `stat_scores_multiple_classes` was deprecated in v1.2.0 in favor of"
+        " `from pytorch_lightning.metrics.functional import stat_scores`."
+        " It will be removed in v1.3.0", DeprecationWarning
```

Comment:
deprecation should be 1.4 here. we always have 0.2 versions between deprecation and removal.

Reply:
also please check this for any other deprecation you have made here if I missed something.

Reply:
This is the only depreciation, will move it